low-level/framework/sspl_rabbitmq_reinit

```diff
@@ -324,10 +322,10 @@ def _create_user_if_necessary(username, password, virtual_host, product):
         response = _send_command(command)
 
         users = response.split("\n")
-        assert users[0] == 'Listing users ...'
         found_user = False
         for userspec in users[1:-1]:
             user = userspec.split()[0]
+            print(f'User: {user}')
```

Comment:
printing 'users' before loop to all values in one go. loop is breaking on one condition, in which case wont see all configured values

Reply:
Done